src/ui/component/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,52 @@
+import type { CSSProperties, FC } from 'react'
+import classNames from 'classnames'
+import './skeleton.scss'
+
+type SkeletonProps = {
+  width?: number | string
+  height?: number | string
+  fontSize?: number | string
+  variant?: 'text' | 'rectangular' | 'rounded' | 'circular'
+  className?: string
+  style?: CSSProperties
+  count?: number
```

Comment:
What's the purpose of those props?
IMHO: 
- `className`: not needed because parent must control the rendering
- `style`: not needed for now, please keep KISS
- `count`: I do not expect this component to map any element.. The purpose of a `Skeleton` component is to render a `Sleleton` component with some style props. I don't want to have any `array` computations if I only want to render one item by one item (e.g a table)

Reply:
`className` and `style` purpose is to customise the skeleton element.
`count`'s purpose is to avoid to display several similar skeleton in a row.
We can remove all of them.